qcow.c

```diff
@@ -1004,7 +1005,7 @@ static uint64_t qcow2_block_alloc(struct qcow_state *s, size_t size)
 	s->first_free_cluster = cluster + s->cluster_size;
 	// this causes a nasty loop
 	// qcow2_set_refcount(s, cluster, 1);
-	tcmu_dbg("  allocating cluster %d\n", cluster / s->cluster_size);
+	tcmu_dbg("  allocating cluster %"PRIu64"\n", cluster / s->cluster_size);
```

Comment:
cluster_size is a "unsigned int", so it should be "%u" right? If that is correct, I can just fix it up for you when I merge. No need to respin your patches.

Reply:
The printf argument is cluster / s->cluster_size, and cluster is uint64_t.